crates/wasmi/src/store.rs

```diff
@@ -67,6 +68,22 @@ impl StoreIdx {
 /// A stored entity.
 pub type Stored<Idx> = GuardedEntity<StoreIdx, Idx>;
 
+pub struct ResourceLimiterRef<'a>(pub(crate) Option<&'a mut (dyn crate::ResourceLimiter)>);
+impl<'a> core::fmt::Debug for ResourceLimiterRef<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "ResourceLimiterRef(...)")
+    }
+}
+
+pub(crate) struct ResourceLimiterQuery<T>(
+    pub(crate) Box<dyn FnMut(&mut T) -> &mut (dyn crate::ResourceLimiter) + Send + Sync>,
```

Comment:
From what I see the `pub(crate)` visibility modifier is not used and can be removed. Probably an oversight.

Reply:
Yeah I was calling it from outside the module at one point while I was moving stuff around. Looks superfluous now.